combustion-android-ble/src/main/java/inc/combustion/framework/ble/NetworkManager.kt

```diff
@@ -163,6 +164,9 @@ internal class NetworkManager(
             mutableDeviceProximityFlow = MutableSharedFlow(
                 FLOW_CONFIG_REPLAY, FLOW_CONFIG_BUFFER, BufferOverflow.SUSPEND
             ),
+            mutableGenericNodeRequestFlow = MutableSharedFlow(
+                FLOW_CONFIG_REPLAY, FLOW_CONFIG_BUFFER, BufferOverflow.SUSPEND
```

Comment:
There might be unintended consequences of replaying old messages from this flow.  I also wonder if should use drop oldest instead of suspend, if the messages over meatnet aren't reliable in the first place.

fwiw -- do we need to replay and suspend on the proximity updates?

Reply:
What you are suggesting makes sense to me. I'll make those changes at least to the `mutableGenericNodeRequestFlow`, I'm going to hold off on the proximity changes.